variant_calling_benchmarks/joint_caller/process_results.py

```diff
@@ -128,7 +135,7 @@ def parse_mixture_likelihoods(strings):
 
 def expand_sample_info_columns_one_row(full_row, result):
     sample_info = full_row['sample_info']
-    for (sample, info) in sample_info.iteritems():
+    for (sample, info) in six.iteritems(sample_info):
```

Comment:
Cool, yea I wasn't sure where `iteritems` was just a random choice vs important so just converted those to `six.`